packages/composites/vesper/src/adapter.ts

```diff
@@ -0,0 +1,27 @@
+import { Validator } from '@chainlink/ea-bootstrap'
+import { ExecuteFactory, ExecuteWithConfig } from '@chainlink/types'
+import { Config, makeConfig } from './config'
+import { ethers } from 'ethers'
+import { getTokenAllocations } from './tvl'
+import * as TokenAllocation from '@chainlink/token-allocation-adapter'
+
+const customParams = {
+  controller: true,
```

Comment:
Instead of requiring the controller for this adapter, we should be able to hard-code it into the config. Vesper said that it should never change.